content/guides/tutorials/server/requirements/index.md

```diff
@@ -53,23 +55,24 @@ The following are required for DNN Platform 9.0 and DNN Evoq 9.0.
 
 ## Developer IDEs
 
-For creating modules:
+For creating extensions:
 
-*   Visual Studio 2013
-*   Visual Studio 2015
 *   Visual Studio 2017
 *   Visual Studio 2019
 
+> [!Note]
+> Starting with DNN 9.4.0, .NET Framework 4.7.2+ is required, however, it is still possible to use older versions of Visual Studio, and .NET Framework for extension development.  The minimum version for extension development is dependent upon the targeted DNN Minimum Version.
 
-For contributing to the DNN Platform:
+For contributing to DNN Platform:
 
-*   Visual Studio 2015
 *   Visual Studio 2017
 *   Visual Studio 2019
 
+
 ## Recommended Environment
 
 *   Windows Server 2012 R2
```

Comment:
Shouldn't we recommend using latest version (2019)?

Reply:
I updated this, as actually really the latest version possible is the best recommendation